### PR TITLE
Dependency Updates

### PR DESCRIPTION
**Pull request by jakobbraun**
Dependency Updates

### CODE REVIEW

File: doc/changes/changes_2.0.1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#Virtual Schema for BigQuery 2.0.1, released 2021-11-23
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_2.0.1.md:1]:
```suggestion
# Virtual Schema for BigQuery 2.0.1, released 2021-11-23
```

File: doc/changes/changes_2.0.1.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Code name: Dependency Updates
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_2.0.1.md:3]: Do you want to add a version here?

**Review by AnastasiiaSergienko — APPROVED** [review]
Approved with minor suggestions

File: doc/changes/changes_2.0.1.md (hunk 2 above)
Comment (jakobbraun) [reply doc/changes/changes_2.0.1.md:3]: no. I'll add it only to the GitHub release. Here it would be doubled.

**Pull request merged**